guides/v2.2/cloud/project/project-start.md

```diff
@@ -60,61 +47,96 @@ In Integration, Staging, and Production, *only* the following directories are wr
 
-### Deploy logs {#deploy-log}
+### Deploy logs
 
 You can review these logs via SSH into the environment. Change to the directories listed below to review the logs.
 
 Logs from the deploy hook are located on the server in the following locations:
 
-*	Integration: `/var/log/deploy.log`
-*	Staging: `/var/log/platform/<project ID>/deploy.log`
-*	Production: `/var/log/platform/{1|2|3}.<project ID>/deploy.log`
+-  **Integration**: `/var/log/deploy.log`
+-  **Staging**: `/var/log/platform/<project-ID>_stg/post_deploy.log`
+-  **Production**: `/var/log/platform/<node-{1|2|3}>.<project-ID>/post_deploy.log`
+
+The value of `<project-ID>` depends on the project ID and whether the environment is Staging or Production. For example, with a project ID of `yw1unoukjcawe`, the Staging environment user is `yw1unoukjcawe_stg` and the Production environment user is `yw1unoukjcawe`. Using that example, the deploy log is located at `/var/log/platform/yw1unoukjcawe_stg/post_deploy.log`.
+
+Pro projects have a three-node structure. Each node contains logs with specific information for that node. For example, on the Production environment for the `yw1unoukjcawe` project, the deploy logs have the following locations:
+
+-  **Node 1**: `/var/log/platform/1.yw1unoukjcawe/post_deploy.log`
+-  **Node 2**: `/var/log/platform/2.yw1unoukjcawe/post_deploy.log`
+-  **Node 3**: `/var/log/platform/3.yw1unoukjcawe/post_deploy.log`
+
+The log for each deployment appends to this file. Check the timestamps on log entries to verify and locate the logs you want for a specific deployment. The following is a condensed example of log output that you can use for troubleshooting:
+
+```terminal
+Re-deploying environment project-integration-ID
+  Executing post deploy hook for service `mymagento`
+    [2019-01-03 19:44:11] NOTICE: Starting post-deploy.  
+    [2019-01-03 19:44:11] INFO: Validating configuration  
+    [2019-01-03 19:44:11] INFO: End of validation  
+    [2019-01-03 19:44:11] INFO: Enable cron  
+    [2019-01-03 19:44:11] INFO: Create backup of important files.  
+    [2019-01-03 19:44:11] INFO: Backup /app/app/etc/env.php.bak for /app/app/etc/env.php was created.  
+    [2019-01-03 19:44:11] INFO: Backup /app/app/etc/config.php.bak for /app/app/etc/config.php was created.  
+    [2019-01-03 19:44:11] INFO: php ./bin/magento cache:flush --ansi --no-interaction   
+    [2019-01-03 19:44:32] INFO: Warming up failed: http://integration-id-project.us.magentosite.cloud/
+    [2019-01-03 19:44:32] NOTICE: Post-deploy is complete.  
+```
+{: .no-copy}
 
-The value of `<project ID>` depends on the project ID and whether the environment is Staging or Production. For example, with a project ID of `yw1unoukjcawe`, the Staging environment user is `yw1unoukjcawe_stg` and the Production environment user is `yw1unoukjcawe`.
+The deploy log contains start and stop messages for each hook, such as `Start deploy.` and `Deployment complete.`.
```

Comment:
the periods in the message examples seem like punctuation mistakes -- consider leaving the period off in the message for easier reading, or rephrase